<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_sp</class>
 <widget class="QWidget" name="Form_sp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pb_smooth">
       <property name="text">
        <string>图像平滑</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_sharp">
       <property name="text">
        <string>图像锐化</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
